Thu Sep 22 19:56:59 2022 - INFO  - Start to prepare the test environment.
ln: failed to create symbolic link '/etc/raddb/mods-enabled/counter': No such file or directory
Failed to start radiusd.service: Unit radiusd.service not found.
Thu Sep 22 19:57:57 2022 - INFO  - End to prepare the test environment.
Thu Sep 22 19:57:58 2022 - INFO  - Start to run test.
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 35: rad_counter: command not found
Thu Sep 22 19:57:59 2022 - ERROR - rad_counter --user execution failed.
Thu Sep 22 19:57:59 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 36
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 37: rad_counter: command not found
Thu Sep 22 19:58:00 2022 - ERROR - rad_counter --match execution failed.
Thu Sep 22 19:58:01 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 38
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 39: rad_counter: command not found
Thu Sep 22 19:58:01 2022 - ERROR - rad_counter --reset execution failed.
Thu Sep 22 19:58:02 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 40
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 41: rad_counter: command not found
Thu Sep 22 19:58:03 2022 - ERROR - rad_counter --help execution failed.
Thu Sep 22 19:58:04 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 42
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 43: rad_counter: command not found
Thu Sep 22 19:58:04 2022 - ERROR - rad_counter --hours execution failed.
Thu Sep 22 19:58:05 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 44
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 45: rad_counter: command not found
Thu Sep 22 19:58:06 2022 - ERROR - rad_counter --minutes execution failed.
Thu Sep 22 19:58:07 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 46
oe_test_freeradius_freeradius-utils_rad_counter.sh: line 47: rad_counter: command not found
Thu Sep 22 19:58:07 2022 - ERROR - rad_counter --seconds execution failed.
Thu Sep 22 19:58:08 2022 - ERROR - oe_test_freeradius_freeradius-utils_rad_counter.sh line 48
Thu Sep 22 19:58:09 2022 - INFO  - End to run test.
Thu Sep 22 19:58:10 2022 - ERROR - failed to execute the case.
Thu Sep 22 19:58:10 2022 - INFO  - Start to restore the test environment.
Failed to stop radiusd.service: Unit radiusd.service not loaded.
Traceback (most recent call last):
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 0:09:58 ago on Thu 22 Sep 2022 07:47:54 PM CST.\nError: \n Problem: conflicting requests\n  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
Thu Sep 22 19:58:14 2022 - INFO  - End to restore the test environment.
